core/modules/system/config/system.core.json

```diff
@@ -54,6 +54,9 @@
     "site_slogan": "",
     "theme_default": "stark",
     "theme_debug": 0,
+    "transliteration_file_lowercase": 1,
+    "transliteration_file_uploads": 1,
+    "transliteration_file_uploads_display_name": 1,
```

Comment:
These variables rightly started with "transliteration_" when provided by transliteration.module. For core, would `file_transliterate_*` be more appropriate? Then it would be grouped with the other related file settings.